### DISCUSSION TITLE
PCB layout ?

### DISCUSSION THREAD

**Discussion by z34pec** [Edzelf/ESP32Radio-V2#138] [General]
There is a pcb layout in edzelf's documents that is very interesting. I order it a year ago but did not pay attention to it till these days. Yesterday when I checked it, I made a notice that pcb's layout of esp32 pins is not compatible with existing versions of esp32 available on the market. Did somebody managed to use it properly or not ? If not, is it possible to be removed from documets so it does confuse users ? Also naked ninja's site is down, does somebody have their gerber files for pcb ? I know there is a pcb layout here with smb components, but personally I do not like it so much.
![1](https://github.com/Edzelf/ESP32Radio-V2/assets/133197383/734a381a-b4ff-478a-8e58-894b28b8eabe)

**Comment by Edzelf** [comment]
A new PCB is under development.  The print will be produced by PCBWay.  It will be available at PCBWay in a view weeks.

**Comment by chainsawdave** [comment]
Hi After months of trying on my own, with a bit of help from Ed, I have got it working, I used the new board from PCB Way and Ed's new instructions. I'll say it's pretty intuitive with Ed's detailed instructions. Some of the streams are a little unstable and I've not managed to get the screen and encoder working but I got sound and it was very good. I went through several ESP32's I don't know why but I think I got a lot of duds.
